src/EluvioLive.js

```diff
@@ -1158,17 +1291,35 @@ Lookup NFT: https://wallet.contentfabric.io/lookup/`; */
       },
     ];
 
+    // Insert rarity
+    for (const i in attrs) {
+      if (rarity && rarity[attrs[i].trait_type]) {
+        let r = rarity[attrs[i].trait_type];
+        attrs[i].rarity = r[attrs[i].value] + "/" + r.total;
```

Comment:
Hmm, I really prefer the denominator to be the number minted. Using just the `trait_type` total is confusing.

Reply:
Ok - I changed the denominator to total supply.  That's better